Composer/packages/client/src/pages/publish/BotStatusList.tsx

```diff
@@ -32,25 +33,35 @@ const copiedCalloutStyles = {
 };
 
 type SkillManifestUrlFieldProps = {
-  url: string;
+  urls: string[];
 };
 
-const SkillManifestUrlField = ({ url }: SkillManifestUrlFieldProps) => {
-  const { isCopiedToClipboard, copyTextToClipboard, resetIsCopiedToClipboard } = useCopyToClipboard(url);
+const SkillManifestUrlField = ({ urls }: SkillManifestUrlFieldProps) => {
+  const { isCopiedToClipboard, copyTextToClipboard, resetIsCopiedToClipboard } = useCopyToClipboard(urls[0]);
 
   const calloutTarget = useRef<HTMLElement>();
+  const clipUrl = (url) => url.replace('azurewebsites.net/manifests/', 'azureweb...');
```

Comment:
is this based on a design? Why not use browser's native ellipsis based on the available space?

Reply:
It is based on design, the url is like `https://www.xxx.com/azureweb...manifest.json` in design